source/stylesheets/site.css.scss

```diff
@@ -55,7 +55,6 @@ $small-max-width: 767px;
 
 body {
   -webkit-font-smoothing: antialiased;
-  background-color: rgba(250,235,232,0.36); //white is too white in contrast to ember orange
```

Comment:
Can you confirm that it's one of these colors from the branding guidelines? 
![image](https://user-images.githubusercontent.com/4587451/47521942-ef1acc80-d859-11e8-8a1d-93dba185132e.png)